### DISCUSSION TITLE
Where do we go for plugins?

### DISCUSSION THREAD

**Discussion by Revenger** [Pryaxis/TShock#1981] [General]
As the forum was locked and is retired wheres a place listing all current and working plugins for TShock?

It seems theres no central place now where to browes and find new plugins and those where the projects are still active.Looking online seems most plugins are now labled as abandoned and there would be varied compatavility with the latest version..

**Comment by madfunny** [comment]
Yeah, I'm looking for plugins, like the "house" one mostly, my fav plugin.

Closing the forums with the plugin list was the single worst idea. I saw the post "needs at least x amount of votes or we close it anyways" Didn't even give anyone a chance.

so, Folks, Where do we find plugins now?

**Comment by tanpro260196** [comment]
I also think closing the forum was not really a good idea after all. The forum was not active because the game itself was not active.

Now it's active again. We're gonna need more than just github discussion and a telegram group chat.

**Comment by hakusaro** [comment]
There is a central place to look for plugins: https://github.com/Pryaxis/Plugins. This is actually listed in the `README` on this repo.

cc @tanpro260196, @madfunny, @Revenger

**Reply by Revenger** [reply]
Only thing with that is theres currently not many in the list and last I checked only had like 4 or 5 plugins so a couple were added it seems.

**Reply by hakusaro** [reply]
It takes a long time for people to update.
